tests/cpp_unit_tests/test_common.cpp

```diff
@@ -0,0 +1,14 @@
+// SPDX-FileCopyrightText: Contributors to the Power Grid Model project <powergridmodel@lfenergy.org>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#include <power_grid_model/common/common.hpp>
+
+namespace power_grid_model {
+namespace {
+static_assert(symmetry_tag<symmetric_t>);
+static_assert(symmetry_tag<symmetric_t>);
+static_assert(is_symmetric<symmetric_t>);
+static_assert(!is_symmetric<asymmetric_t>);
```

Comment:
should i also add a test here for `other_symmetry_t`?

Reply:
Just do it.